### PR TITLE
Ansible 2.6

### LINKED ISSUE #761: vulnerable version of Ansible
Looks like there is a [security issue](https://nvd.nist.gov/vuln/detail/CVE-2019-3828) with Ansible <2.5.15.

We should upgrade the requirements.txt after testing that it works with an up-to-date version of Ansible. 

Any takers?

### PR DESCRIPTION
**Pull request by adborden**
Fixes #761

### CODE REVIEW

File: roles/git/files/gitolite.pub [hunk 1]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDgepEgS+Z+UuG2tSQzgn546V4M6mJtk+AfD3pgvRsj6rzAw4tyhSPGeJsXdlNkNi54cJiRzOZNuVXKn2ojc1y+cD88vfSnnEc+/DkfW0cDYowSgzIxfOaYzX5qj0FXh91eaFmg6+yx2AsHXXtWaesqM/4WpQq8d0VEMqhrelIjgBsLQONMoaj7adFARuhlMMbjqvW5aKZeuB8CGW5VprebwUPzRfEhvimO/IZ9+Z08qtxTXtWiUX9G/OeZ4+ddBkkO0XHFwCbVDa4AM/I6jo18hKz5EE9q4ZS7XOKVHElqbK/fg4bHBJqS5VM8QPMBEi+uGFjksSHxhWDWoMw62YC5 adborden@cano
```
Comment (Titan-C) [inline roles/git/files/gitolite.pub]: over-commited
Does not belong in repo

Comment (adborden) [reply roles/git/files/gitolite.pub]: Oops, I will remove this and rebase.

**Comment by adborden** [comment]
Rebased, thanks!

**Comment by adborden** [comment]
Sorry for the extra commits. Originally I wasn't going to address the deprecation warnings, but then decided to just take care of them now. This is ready for review.

File: roles/common/tasks/main.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@
     - dependencies
 
 - name: Install necessities and nice-to-haves
-  apt: pkg={{ item }} state=installed
+  apt: pkg={{ item }} state=present
```
Comment (neuhaus) [inline roles/common/tasks/main.yml]: Is it pkg or name?

File: roles/owncloud/tasks/main.yml [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
+      OwnCloud is not supported in this distribution release
       https://github.com/sovereign/sovereign/issues/765
-  when: ansible_distribution_release == "jessie"
+  when: ansible_distribution_release in ["jessie", "trusty"]
 
-- include: owncloud.yml tags=owncloud
+- import_tasks: owncloud.yml
+  tags: owncloud
   # https://github.com/sovereign/sovereign/issues/765
   when: ansible_distribution_release != "jessie"
```
Comment (neuhaus) [inline roles/owncloud/tasks/main.yml]: Forgot trusty here

**Comment by neuhaus** [comment]
Great work! I've added some comments.

File: roles/common/tasks/main.yml (hunk 2 above)
Comment (adborden) [reply roles/common/tasks/main.yml]: I prefer `name`, but pkg and package are valid aliases. pkg seemed consistent throught the playbook, so I left it as is.

File: roles/owncloud/tasks/main.yml (hunk 3 above)
Comment (adborden) [reply roles/owncloud/tasks/main.yml]: Oops, I will add.

File: roles/common/tasks/encfs.yml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 - name: Install encfs & fuse
-  apt: pkg={{ item }} state=installed
+  apt: name={{ item }} state=present
```
Comment (neuhaus) [inline roles/common/tasks/encfs.yml]: I suggest you use pkg here for consistency

File: roles/common/tasks/main.yml (hunk 2 above)
Comment (neuhaus) [reply roles/common/tasks/main.yml]: Ok, i added a comment to the inconsistent one ☝🏽

**Comment by adborden** [comment]
This is ready for another look.

**Comment by adborden** [comment]
:wave: Ansible 2.9 was just released, [making 2.6 end of life](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html) and unsupported. Can we get this merged so we can start on a 2.7 update?

**Comment by adborden** [comment]
:wave: pinging again on this.

Folks are duplicating work unnecessarily in https://github.com/sovereign/sovereign/pull/797 and https://github.com/sovereign/sovereign/pull/798

**Comment by neuhaus** [comment]
Thanks, sorry about the delay, was hoping for more reviews

**Pull request merged**